### PR TITLE
mbed-cli: init at 1.8.3

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/mbed-cli/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, pythonPackages }:
```
Comment (dotlambda) [inline pkgs/development/tools/mbed-cli/default.nix]:
```suggestion
{ lib, python3Packages }:
```

File: pkgs/development/tools/mbed-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  version = "1.8.3";
+
+  src = pythonPackages.fetchPypi {
+    inherit pname version;
+    sha256 = "04vn2v0d7y3vmm8cswzvn2z85balgp3095n5flvgf3r60fdlhlmp";
+  };
+
+  doCheck = false; # no tests available in Pypi
+
+  meta = with stdenv.lib; {
```
Comment (dotlambda) [inline pkgs/development/tools/mbed-cli/default.nix]:
```suggestion
  meta = with lib; {
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -11619,6 +11619,8 @@ in
 
   matio = callPackage ../development/libraries/matio { };
 
+  mbed-cli = callPackage ../development/tools/mbed-cli {  };
```
Comment (dotlambda) [inline pkgs/top-level/all-packages.nix]:
```suggestion
  mbed-cli = callPackage ../development/tools/mbed-cli { };
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 3 above)
Comment (dotlambda) [reply pkgs/top-level/all-packages.nix]: I think you've put this in the wrong category. It should be below `### TOOLS`.

**Comment by rvolosatovs** [comment]
@dotlambda

**Comment by dotlambda** [comment]
@GrahamcOfBorg build mbed-cli

**Comment by dotlambda** [comment]
@rvolosatovs 1.9.0 is out

**Pull request merged**